apache-cassandra-mixin/config.libsonnet

```diff
@@ -15,7 +19,7 @@
     alertsCriticalHighCpuUsage5m: 80,  //percent: emitted metric has range 0-100
     alertsCriticalHighMemoryUsage5m: 80,  //percent: calculated as ratio then multiplied by query
 
-    enableLokiLogs: true,
+    enableLokiLogs: false,
```

Comment:
I think we want logs enabled by default?

Reply:
@schmikei wow, that's just a dum dum mistake from moving too fast. thank you for noticing 😄 . the correct config value should be updated now